app/build.gradle

```diff
@@ -385,6 +385,11 @@ android {
         // MetaStore and AppLab only apply to oculusvr builds.
         if ((store == 'metaStore' || store == "appLab") && !platform.startsWith('oculusvr'))
             variant.setIgnore(true);
+        
+        // Throw an error if a store flavor is built with openxr=false
+        if (store == 'generic' && !project.hasProperty('openxr') || project.getProperty('openxr') == false) {
```

Comment:
Hmm but `generic`is precisely used for non store builds. The point is that store builds must always use `openxr=true`